Just tested this on Windows 10 myself, and it's NOT fully working yet.

While the build succeeds, Mirage 2 does **not** build properly and throws errors (the fact that it reports success despite errors is already a know issue: https://jira.duraspace.org/browse/DS-2458)

The errors seem to occur in the node installation process, as node cannot be downloaded (404) and never ends up being installed. Here's the errors I'm seeing:
```
[INFO] ------ (Mirage2) com.github.eirslett:frontend-maven-plugin:0.0.23:install-node-and-npm
[INFO] Installing node version v6.5.0
[INFO] No proxy was configured, downloading directly
[ERROR] MojoFailureException:
org.apache.maven.plugin.MojoFailureException: Could not download Node.js from: http://nodejs.org/dist/v6.5.0/x64/node.exe: Got error code 404 from the server.

... (After this there are several errors stating "Failed to run task" because "node" cannot be found)...

[INFO] ------ (Mirage2) de.saumya.mojo:gem-maven-plugin:1.0.6:exec
[WARNING] Compass can't find any Sass files to compile.
[WARNING] Is your compass configuration correct?.
[WARNING] If you're trying to start a new project, you have left off the directory argument.
[WARNING] Run "compass -h" to get help.
[ERROR] PluginManagerException:
org.apache.maven.plugin.PluginExecutionException: Execution null of goal de.saumya.mojo:gem-maven-plugin:1.0.6:exec failed: Java returned: 1
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
        at com.soebes.maven.plugins.iterator.ExecutorMojo.executeMojo(ExecutorMojo.java:302)
        at com.soebes.maven.plugins.iterator.ExecutorMojo.execute(ExecutorMojo.java:235)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: Java returned: 1
        at org.apache.tools.ant.taskdefs.Java.execute(Java.java:87)
        at de.saumya.mojo.ruby.script.AntLauncher.doExecute(AntLauncher.java:107)
        at de.saumya.mojo.ruby.script.AntLauncher.executeIn(AntLauncher.java:141)
        at de.saumya.mojo.ruby.script.Script.executeIn(Script.java:108)
        at de.saumya.mojo.gem.ExecMojo.executeWithGems(ExecMojo.java:106)
        at de.saumya.mojo.gem.AbstractGemMojo.executeJRuby(AbstractGemMojo.java:469)
        at de.saumya.mojo.jruby.AbstractJRubyMojo.execute(AbstractJRubyMojo.java:298)
        at de.saumya.mojo.gem.AbstractGemMojo.execute(AbstractGemMojo.java:222)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
        ... 24 more
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,9 +7,12 @@ env:
 
 # Install prerequisites for building Mirage2 more rapidly
 before_install:
-  # Install latest Node.js 0.10.x & print version info
-  - nvm install 0.10
+  # Install Node.js 6.5.0 & print version info
+  - nvm install 6.5.0
   - node --version
+  # Install npm 3.10.8 & print version info
+  - npm install -g npm@3.10.8
+  - npm --version
   # Install Bower
   - npm install -g bower
   # Install Grunt & print version info

--- a/dspace/modules/xmlui-mirage2/pom.xml
+++ b/dspace/modules/xmlui-mirage2/pom.xml
@@ -350,7 +350,7 @@
                         <plugin>
                             <groupId>com.github.eirslett</groupId>
                             <artifactId>frontend-maven-plugin</artifactId>
-                            <version>0.0.23</version>
+                            <version>1.0</version>
                         </plugin>
                     </plugins>
                 </pluginManagement>

